Cargo.lock

```diff
@@ -64,30 +65,30 @@ dependencies = [
 
 [[package]]
 name = "aho-corasick"
-version = "0.7.13"
+version = "0.7.15"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "043164d8ba5c4c3035fec9bbee8647c0261d788f3474306f93bb65901cae0e86"
+checksum = "7404febffaa47dac81aa44dba71523c9d069b1bdc50a77db41195149e17f68e5"
 dependencies = [
  "memchr",
 ]
 
 [[package]]
 name = "akri-shared"
-version = "0.1.10"
+version = "0.1.11"
 dependencies = [
  "anyhow",
  "async-trait",
  "bytes 0.5.6",
  "either",
  "env_logger",
- "futures 0.1.29",
- "futures 0.3.5",
+ "futures 0.1.30",
+ "futures 0.3.12",
  "futures-util",
- "hyper 0.11.27",
- "hyper 0.13.7",
+ "hyper 0.12.35",
```

Comment:
Do we need two different versions of hyper here?

Reply:
shared cargo.toml references 2 different hyper versions and that translates to 2 different versions here. I am not sure why that's the case thou, doesn't seem necessary. I'd be glad to update it.